src/main/java/frc/robot/maps/GastonMap.java

```diff
@@ -0,0 +1,73 @@
+package frc.robot.maps;
+
+import com.chopshop166.chopshoplib.drive.SDSSwerveModule;
+import com.chopshop166.chopshoplib.motors.PIDSparkMax;
+import com.chopshop166.chopshoplib.sensors.PigeonGyro;
+import com.ctre.phoenix.sensors.AbsoluteSensorRange;
+import com.ctre.phoenix.sensors.CANCoder;
+import com.ctre.phoenix.sensors.PigeonIMU;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.interfaces.Gyro;
+
+public class GastonMap extends RobotMap {
+    @Override
+    public DriveMap getDriveMap() {
+
+        // Value taken from CAD as offset from center of module base pulley to center of
+        // robot
+        final double MODULE_OFFSET_XY = 0; // TODO Get this value
```

Comment:
Default the offset to 1 just so we hopefully avoid straight-up crashes. Or even better, if you can get the real size from CAD that would be great.